MadLabs/MadLabs.Hub/Views/Games/Index.cshtml

```diff
@@ -6,20 +6,21 @@
 
 <div class="container container-large pt-2">
     <h2 class="text-center pb-5">Available Games</h2>
-    <div class="row">
+    <div class="card-deck h-100">
         @{
             int count = 0;
         }
         @foreach (var metadata in (List<MadLabs.Hub.ViewModels.GameMetadataViewModel>)Model)
         {
-        <div class="col-lg-6 col-md-12" onclick="location.href='@Url.Action("Play", "Games", new { gameId = count })'">
```

Comment:
you removed col-lg-6 -> that created 2 columns of cards (since bootstrap is based on a grid of "12 units"